src/Composer/Installers/DrupalInstaller.php

```diff
@@ -20,5 +20,6 @@ class DrupalInstaller extends BaseInstaller
         'console'          => 'console/{$name}/',
         'console-language' => 'console/language/{$name}/',
         'config'           => 'config/sync/',
+        'recipe'           => 'recipe/{$name}',
```

Comment:
```suggestion
        'recipe'           => 'recipes/{$name}',
```
For consistency with the other types of Drupal packages.